Check and balances.md

```diff
@@ -11,11 +11,24 @@ In this case. Delijaj is the seller and Nyomi the buyer.
 While testing the trading system on the 3<sup>rd</sup> auction, a seller decided to trad some parts of the project. 
 However, she did not own any part of it but the website let her do it anyway. The parts were at 50 token. A buyer 
 decided to acquire those parts. The exchange was made and the seller finished with a -50 on the project (like a debt).
-This shouldn't happend since the parts must be only managed by the contributors and nobody else.
+This shouldn't happend since the parts must be only managed by the contributors and nobody else, and also to prevent any scam.
 
-## Sponsor/Inversor 
-As the website is today, the trade is at its basic stage. To improve it, we can put a system of sponsor/investor.
-Indeed if we look at the projects, some people will have fantastic ideas but not the means to do it. So to help developping 
-the market and the motivation to creat a projet. In the case that a person doesn't want to continue the project, for any reasons,
-she could sell it in the platforme to keep it going.
-It will creat a real market and viability of the projects.
+## Licence
+I don’t see the point on creating a new licence. I feel like we should use what we already have since creating such a licence 
+would be assembling existing licences. (and some project would have similar stuff). 
+“provisional financial attribution based on share ownership, which itself is based on the contribution & trading trails
+over the whole history of the project.” This is not in the good place for me, it should be shown at the contributor’s 
+profile and thus on how we manage the profiles.
+
+## Encouragement to Share
+In what way?By social media? By sharing with other people you “followed” on the platform?
+I don’t see how other than having a kind of friend list and we already have a way to advise people for projects.
+Since in the whitepaper there isn’t anything written, we could give some ideas.
+
```

Comment:
Interesting you could talk to Thomas about this to see more clearly what he meant and maybe submit a pull request directly to the whitepaper repo. I think it is not just about social media. The gamification and financial aspects are strong motors for encouragement.